.bazelignore

```diff
@@ -0,0 +1 @@
+examples/
```

Comment:
It might not be needed as Bazel should auto-detect it by detecting `WORKSPACE` file.

Reply:
`.bazelignore` also dropped.